simplenote_cli/sncli.py

```diff
@@ -1081,6 +1081,22 @@ def cli_note_create(self, from_stdin, title):
             self.ndb.create_note(content)
             self.sync_notes()
 
+    def cli_note_import(self, from_stdin):
+
+        if from_stdin:
+            raw = ''.join(sys.stdin)
+        else:
+            raw = self.exec_cmd_on_note(None)
```

Comment:
It would definitely be better if it takes a JSON file. I just tried to imitate the behaviour of the create switch, although the create switch can also benefit from taking a text file in some cases.